source/CardGames.py

```diff
@@ -1,6 +1,8 @@
 import random
 import os
 
+from pytest import MonkeyPatch
+
```

Comment:
What is this for? I don't see where it is used in this file. Also if it is pytest, wouldn't it be in a test file?

Reply:
Yeah not too sure what this is for. Perhaps you meant to put it in the test file and forgot to take it out of the CardGames file?